--- pkgIndex.tcl.in.orig    2020-08-28 19:11:50.885601190 +0800
+++ pkgIndex.tcl.in 2020-08-28 00:32:27.000000000 +0800
@@ -2,5 +2,5 @@
 # Tcl package index file
 #
 package ifneeded @PACKAGE_NAME@ @PACKAGE_VERSION@ \
-    "load [file join $dir @PKG_LIB_FILE@];
+    "load [file join @PKG_LIB_FILE@];
      source [list [file join $dir tdom.tcl]]"

--- Makefile.in.orig	2020-08-28 15:40:52.131821382 +0800
+++ Makefile.in	2020-08-28 15:43:13.268040606 +0800
@@ -333,7 +333,7 @@
 
 pkgIndex.tcl-hand:
 	@(echo 'package ifneeded $(PACKAGE_NAME) $(PACKAGE_VERSION) \
-	"load [list [file join $$dir $(PKG_LIB_FILE)]];\
+    "load $(PKG_LIB_FILE);\
          source [list [file join $$dir tdom.tcl]]"'\
 	) > pkgIndex.tcl
 
@@ -450,8 +450,8 @@
 	@$(INSTALL_DATA_DIR) $(DESTDIR)$(pkglibdir)
 	@list='$(lib_BINARIES)'; for p in $$list; do \
 	  if test -f $$p; then \
-	    echo " $(INSTALL_LIBRARY) $$p $(DESTDIR)$(pkglibdir)/$$p"; \
-	    $(INSTALL_LIBRARY) $$p $(DESTDIR)$(pkglibdir)/$$p; \
+	    echo " $(INSTALL_LIBRARY) $$p $(DESTDIR)$(libdir)/$$p"; \
+	    $(INSTALL_LIBRARY) $$p $(DESTDIR)$(libdir)/$$p; \
 	    ext=`echo $$p|sed -e "s/.*\.//"`; \
 	    if test "x$$ext" = "xdll"; then \
 		lib=`basename $$p|sed -e 's/.[^.]*$$//'`.lib; \
